src/config/bookconfig.rs

```diff
@@ -17,6 +18,8 @@ pub struct BookConfig {
     multilingual: bool,
     indent_spaces: i32,
 
+    playpen_config: PlaypenConfig,
```

Comment:
It might be something for refactoring on another day. Some features of playpen might be shared with another renderers (like styling based on class https://github.com/rust-lang/book/issues/55) but I might be reaching here.